docs-next/pages/guides/cli.mdx

```diff
@@ -2,81 +2,188 @@ import { Markdown } from '../../components/Page';
 
 # Command Line
 
-Keystone provides a command line interface (CLI) for running your Keystone system.
-The command `keystone-next` lets you control how you build and run your system, and how your migrate and seed the data in your database.
+Keystone's command line interface (CLI) has been designed to help you develop, build, and deploy your Keystone project.
+
+Using the `keystone-next` command you can start the dev process, build and run your app in production, and control how you migrate the structure of your database as your schema changes.
+
+### Usage
+
+```text
+$ keystone-next [command]
 
-```
-Usage
-  $ keystone-next [command]
 Commands
-  Run
-    dev           (default) start the project in development mode
-    start         start the project in production mode
-  Build
-    build         build the project (must be done before using start)
-  Migrate (Prisma only)
-    reset         reset the database (this will drop all data!)
-    generate      generate a migration
-    deploy        deploy all migrations
+  dev           start the project in development mode (default)
+  postinstall   generate client APIs and types (optional)
+  build         build the project (must be done before using start)
+  start         start the project in production mode
+  prisma        run Prisma CLI commands safely
```

Comment:
@mitchellhamilton We should update `cli.ts` to be consistent with this. At the moment someone running `keystone-next help` will see something different to this.